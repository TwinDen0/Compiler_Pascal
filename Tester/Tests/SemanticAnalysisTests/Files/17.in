var
x : real;
begin
x := 5 + 1.1;
end.