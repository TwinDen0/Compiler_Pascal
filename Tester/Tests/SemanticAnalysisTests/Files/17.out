program
├─── var
│    └─── real
│         └─── x
├─── begin
│    └─── :=
│         ├─── x
│         └─── +
│              ├─── real
│              │    └─── 5
│              └─── 1,1
└─── end



Symbol Tables:
builtins:
	integer: SymInteger
	real: SymReal
	string: SymString
	write: SymProc
	read: SymProc
	exit: SymProc
globals:
	x: SymVar
